lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -562,7 +527,9 @@ def define
         option :less_than_a_year_other
         option :more_than_a_year
 
-        save_input_as :how_long_abroad_jsa
+        on_response do |response|
+          self.how_long_abroad_jsa = response
```

Comment:
for consistency - save state to a calculator.

Reply:
I can't find where this parameter is used so I'll remove it.